en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-example.md

```diff
@@ -0,0 +1,419 @@
+    <img src="../../../../assets/img/connectors/download-zip.png" width="200" alt="Download ZIP">
+</a>
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}   
+
+## Testing
+
+Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here] (https://curl.haxx.se/download.html).
+
+1. Retrieve stock volume details from the Stockquote back-end service.
+ 
+   **Sample request 1**
+
+    ```
+     curl -v GET "http://localhost:8290/stockquote/view/WSO2" -H "Content-Type:application/json"    
+    ```
+
+   **Expected Response**
+    
+     ```json
+     {
+         "Envelope": {
+             "Body": {
+                 "getQuoteResponse": {
+                     "change": -2.86843917118114,
+                     "earnings": -8.540305401672558,
+                     "high": -176.67958828498735,
+                     "last": 177.66987465262923,
+                     "low": -176.30898912339075,
+                     "marketCap": 56495579.98178506,
+                     "name": "WSO2 Company",
+                     "open": 185.62740369461244,
+                     "peRatio": 24.341353665128693,
+                     "percentageChange": -1.4930577008849097,
+                     "prevClose": 192.11844053187397,
+                     "symbol": "WSO2",
+                     "volume": 7791
+                 }
+             }
+         }
+     }
+     ```
+     
+   **Sample request 2**
+     
+     ```
+      curl -v GET "http://localhost:8290/stockquote/view/IBM" -H "Content-Type:application/json"    
+     ```
+     
+   **Expected Response**
+         
+     ```json
+     {
+         "Envelope": {
+             "Body": {
+                 "getQuoteResponse": {
+                     "change": -2.86843917118114,
+                     "earnings": -8.540305401672558,
+                     "high": -176.67958828498735,
+                     "last": 177.66987465262923,
+                     "low": -176.30898912339075,
+                     "marketCap": 56495579.98178506,
+                     "name": "IBM Company",
+                     "open": 185.62740369461244,
+                     "peRatio": 24.341353665128693,
+                     "percentageChange": -1.4930577008849097,
+                     "prevClose": 192.11844053187397,
+                     "symbol": "IBM",
+                     "volume": 7791
+                 }
+             }
+         }
+     }      
+        
+     ```
+   **Inserted hash map can check using `redis-cli`**  
+   
+     Log in to the `redis-cli` and execute `HGETALL StockVolume` command to retrieve inserted hash map details.
+   
+     ```
+     127.0.0.1:6379> HGETALL StockVolume
+     1) "IBM"
+     2) "7791"
+     3) "WSO2"
+     4) "7791"
+     127.0.0.1:6379>
+     ```
+2. Retrieve all stock volume details from the redis server.
```

Comment:
Does the formatting render correctly in the doc site here?

Reply:
Yes